Anytype/Sources/PresentationLayer/TextEditor/Set/Views/Popups/ViewTypes/SetViewTypesPickerViewModel.swift

```diff
@@ -99,11 +99,10 @@ final class SetViewTypesPickerViewModel: ObservableObject {
     
     private func createView() {
         let name = name.isEmpty ? Loc.SetViewTypesPicker.Settings.Textfield.Placeholder.untitled : name
-        Task {
-            try await dataviewService.createView(
-                DataviewView.created(with: name, type: selectedType),
-                source: source
-            )
+        Task { [weak self] in
+            guard let self else { return }
```

Comment:
you capture self immediately when task started for all async calls

Reply:
you mean we don't nee to make this `guard let self else { return }`?

Reply:
yep

Reply:
Removed in this case